share/lib/python/neuron/__init__.py

```diff
@@ -1331,12 +1331,36 @@ def nrnpy_vec_math(op, flag, arg1, arg2=None):
 
   return NotImplemented
 
+def _nrnpy_rvp_pyobj_callback(f):
+  # unless f is an rxd variable, we return it directly
+  f_type = str(type(f))
+  if f_type not in ("<class 'neuron.rxd.species.SpeciesOnRegion'>", "<class 'neuron.rxd.species.Species'>"):
```

Comment:
Should we include "<class 'neuron.rxd.species.State'>", "<class 'neuron.rxd.species.Parameter'>" and "<class 'neuron.rxd.species.ParameterOnRegion'>"?